### PR TITLE
Update agave.js

### PR DESCRIPTION
**Pull request by Luigy-Lemon**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by 0xngmi** [comment]
can you add some code to also include old tvl?

**Comment by llamatester** [comment]
The adapter at projects/agave.js exports TVL: 
        
 
 
    xdai-staking              564.08 k
    xdai-borrowed             2.31 M
    xdai                      7.27 M
    staking                   564.08 k
    borrowed                  2.31 M
    
    total                    7.27 M

**Comment by llamatester** [comment]
The adapter at projects/agave.js exports TVL: 
        
 
 
    xdai-staking              555.98 k
    xdai-borrowed             26.40 M
    xdai                      14.29 M
    staking                   555.98 k
    borrowed                  26.40 M
    
    total                    14.29 M

**Comment by llamatester** [comment]
The adapter at projects/agave.js exports TVL: 
        
 
 
    xdai-staking              557.00 k
    xdai-borrowed             15.52 M
    xdai                      7.17 M
    staking                   557.00 k
    borrowed                  15.52 M
    
    total                    7.17 M

**Comment by 0xngmi** [comment]
btw @g1nt0ki after the hack we need to zero out the borrowing from old version, since all that money is gone

currently we are incorrectly displaying it as borrowed

**Comment by g1nt0ki** [comment]
I was in in two minds about it, okay, setting it to zero

**Comment by llamatester** [comment]
The adapter at projects/agave.js exports TVL: 
        
 
 
    xdai-staking              560.28 k
    xdai                      7.30 M
    xdai-borrowed             2.32 M
    staking                   560.28 k
    borrowed                  2.32 M
    
    total                    7.30 M

**Comment by 0xngmi** [comment]
> I was in in two minds about it

ive been doing this for other lending protocols that got hacked, like cream, but in this case i missed that agave got hacked

i think it makes sense since those funds are basically gone, no longer TVL in any form

File: projects/agave.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = {
     ]),
     borrowed: sdk.util.sumChainTvls([
       aaveChainTvl("xdai", addressesProviderRegistryXDAI, addr => `xdai:${addr}`, [protocolDataHelper], true),
-      aaveChainTvl("xdai", addressesProviderRegistryXDAI_old, addr => `xdai:${addr}`, [protocolDataHelper_old], true),
+      // aaveChainTvl("xdai", addressesProviderRegistryXDAI_old, addr => `xdai:${addr}`, [protocolDataHelper_old], true), // borrowed amount is lost due to attack
```
Comment (0xngmi) [inline projects/agave.js:27]: @g1nt0ki i would use a function that count is before the hack but not after, that way it can be refilled

Comment (g1nt0ki) [reply projects/agave.js:27]: fixing it

**Pull request merged**